easybuild/tools/package/utilities.py

```diff
@@ -173,20 +193,20 @@ def __init__(self):
         if sel_pns in avail_pns:
             self.pns = avail_pns[sel_pns]()
         else:
-            raise EasyBuildError("Selected package naming scheme %s could not be found in %s",
-                                 sel_pns, avail_pns.keys())
+            raise EasyBuildError("Selected package naming scheme %s could not "
+                                 "be found in %s", sel_pns, avail_pns.keys())
```

Comment:
our code style allows lines up until 120 chars, the default in pep8/pylint is 80 I think (which is a nightmare imho)